CONTRIBUTING.md

```diff
@@ -82,7 +82,7 @@ To set up `PACKAGENAME` for local development:
    is broken on your local system:
 
    ```
-   pytest PACKAGENAME
+   pytest
```

Comment:
I don't think the PACKAGENAME is needed here. When I run tests locally, pytest is all I type.